docs/api/actions-and-connectors.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[actions-and-connectors-api]]
+== Actions and Connectors APIs
+
+Manage your {kib} actions and connectors.
+
+The following {kib} actions APIs are available: 
+
+* <<actions-and-connectors-api-get, Get action API>> to retrieve a single {kib} action by ID
+
+* <<actions-and-connectors-api-get-all, Get all actions API>> to retrieve all {kib} actions
+
+* <<actions-and-connectors-api-list, List all action types API>> to retrieve a list of all {kib} action types
+
+* <<actions-and-connectors-api-create, Create action API>> to create {kib} actions
+
+* <<actions-and-connectors-api-update, Update action API>> to update the attributes for an existing {kib} action
+
+* <<actions-and-connectors-api-execute, Execute action API>> to execute a {kib} action by ID
+
+* <<actions-and-connectors-api-delete, Delete action API>> to remove a {kib} action by ID
+
+=== Action types
```

Comment:
It looks like we already document action and connectors types here - https://www.elastic.co/guide/en/kibana/current/action-types.html

I would remove lines 22-73, then replace it with the following:

For information about the actions and connectors that {kib} supports, refer to <<action-types,Action and connector types>>.